src/LTDTree/LTDTree.c

```diff
@@ -27,29 +27,19 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
  ********************************************************************PGR-GNU*/
 
-
-/******************************************************************************/
-/*                          MODIFY AS NEEDED                                  */
-
 #include <stdbool.h>
 
 #include "c_common/postgres_connection.h"
-#include "utils/array.h"
-#include "catalog/pg_type.h"
-#include "utils/lsyscache.h"
-
-#ifndef INT8ARRAYOID
-#define INT8ARRAYOID    1016
-#endif
 
 #include "c_common/debug_macro.h"
 #include "c_common/e_report.h"
 #include "c_common/time_msg.h"
 #include "c_types/pgr_ltdtree_rt.h"
 #include "c_common/edges_input.h"
-#include "c_common/arrays_input.h"
+
 #include "drivers/LTDTree/LTDTree_driver.h"
 
+
 PGDLLEXPORT Datum _pgr_lengauer_tarjan_dominator_tree (PG_FUNCTION_ARGS);
 PG_FUNCTION_INFO_V1(_pgr_lengauer_tarjan_dominator_tree );
```

Comment:
We use camel case
So your function names should be:
`_pgr_lengauertarjandominatortree` in the C file
and `_pgr_lengauerTarjanDominatorTree`  `pgr_lengauerTarjanDominatorTree` on the SQL files

Reply:
I need to fix the name in all pgTap and docs where I used ```_pgr_lengauer_tarjan_dominator_tree```.

Reply:
This needs to be done

Reply:
done ma'am